### PR TITLE
Proposal for streaming framework.

### PR DESCRIPTION
**Pull request by dasmb**
I am excited about the prospect of no code, but am afraid that even with its extraordinarily low latencies on general purpose thoughtware that it will not be performant enough for certain hype-bubbles within the industry.

Thus I am proposing a streaming framework. It is somewhat protocol agnostic. I would prefer no handshake, no authorization and no acknowledgement but let's face it, enterprises are going to want TLS and OAUTH 2.

### CODE REVIEW

**Comment by tedmiston** [comment]
How do you feel about adding micro-batching support?

File: STREAMING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Streaming No Code
+Streaming no code is almost as easy as standard no code. You're still not doing any coding, but now you also
+adopt the policy of not going back to things you previously failed to code and adding code to them and trying
+not to dwell on other things you've written no code for in the past, all while being careful not to get in the 
+way of other people who are also trying not to do anything.
+
+Streaming allows you to write and deploy no code to systems in real time, where real time is defined as "fast enough
+that your boss never has to dog you about it being slow." Streaming no code is actually beyond real time -- it's done in
+no time, flat.
```
Comment (agilgur5) [inline STREAMING.md]: So does this mean streaming no code is faster than lightspeed? Sounds like it implements quantum entanglement / "spooky action at a distance" between systems under the hood -- this is really revolutionary, bleeding-edge tech! I think the same underlying principle could make for a visionary geographically distributed system

Comment (dasmb) [reply STREAMING.md]: It is faster than light, but it is not faster than no-light, so we're good. Basically the laws of physics couldn't give two figs.

**Comment by dasmb** [comment]
Closing, as merging would be a no-op.

**Pull request closed without merging**